src/cmd/property.go

```diff
@@ -11,6 +11,73 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var examplePropertyCmd = &cobra.Command{
+	Use:   "property",
+	Short: "Example Property",
+	Long:  `Example Property`,
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println(getExample[opslevel.PropertyInput]())
+	},
+}
+
+var getPropertyCmd = &cobra.Command{
+	Use:        "property",
+	Short:      "Get details about an assigned property",
+	Long:       `Get details about an assigned property`,
+	Example:    `opslevel get property owner-alias property-id`,
+	Args:       cobra.ExactArgs(2),
+	ArgAliases: []string{"ID", "ALIAS"},
+	Run: func(cmd *cobra.Command, args []string) {
+		ownerId := args[0]
+		propertyId := args[1]
+
+		result, err := getClientGQL().GetProperty(ownerId, propertyId)
+		cobra.CheckErr(err)
+
+		if isYamlOutput() {
+			common.YamlPrint(result)
+		} else {
+			common.PrettyPrint(result)
+		}
+	},
+}
+
+var assignPropertyCmd = &cobra.Command{
+	Use:   "property",
+	Short: "Assign a Property",
+	Long:  `Assign a Property`,
+	Example: fmt.Sprintf(`
+cat << EOF | opslevel assign property-definition -f -
+%s
+EOF`, getYaml[opslevel.PropertyInput]()),
+	Run: func(cmd *cobra.Command, args []string) {
+		input, err := readResourceInput[opslevel.PropertyInput]()
+		cobra.CheckErr(err)
+		newProperty, err := getClientGQL().PropertyAssign(*input)
+		cobra.CheckErr(err)
+
+		fmt.Println(newProperty.Definition.Id)
```

Comment:
```suggestion
		fmt.Printf("assigned property '%s' on '%s'\n", newProperty.Definition.Id, newProperty.Owner.Id)
```

Suggested output is more convenient (contains parameters needed to `get property`) and is consistent with the output of `unassign`